tests/generic_config_updater/gu_common_test.py

```diff
@@ -258,6 +258,35 @@ def test_remove_empty_tables__multiple_empty_tables__returns_config_without_empt
         # Assert
         self.assertDictEqual({"any_table": {"key": "value"}}, actual)
 
+    def test_create_sonic_yang_with_loaded_models__creates_new_sonic_yang_every_call(self):
```

Comment:
>test_create_sonic_yang_with_loaded_models__creates_new_sonic_yang_every_call [](http://example.com/codeflow?start=8&length=76)

You mentioned
> TestPatchSorter took ... Before: 800s

The performance comparison is interesting. Could you explain what happens in 800s? How can we repro your result by unit test?

Reply:
updated pr description with profiling data for a specific case instead of all cases in `TestPatchSorter`